projects/migrations/0006_team_emoji.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.6 on 2025-02-27 07:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0005_team_leads_team_members_alter_project_managers_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='team',
+            name='emoji',
+            field=models.CharField(blank=True, max_length=3, null=True),
```

Comment:
Why is max length 100 in the model but 3 in the migration?

Reply:
good question